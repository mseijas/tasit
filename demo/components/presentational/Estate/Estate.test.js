import React from "react";
import { shallow } from "enzyme";
import Estate from "./Estate";

describe("Estate", () => {
  it("renders the component", async () => {
    const estate = {
      id: 123,
      name: "Not found",
      img: "https://decentraland.org/images/logo-65f7b27caf.png",
    };
    expect(shallow(<Estate estate={estate} />)).toMatchSnapshot();
  });
});
