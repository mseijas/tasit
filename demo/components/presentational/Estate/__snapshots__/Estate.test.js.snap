// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Estate renders the component 1`] = `
<Fragment>
  <Component
    source={
      Object {
        "uri": "https://decentraland.org/images/logo-65f7b27caf.png",
      }
    }
    style={
      Object {
        "height": 266.8,
        "width": 450,
      }
    }
  />
  <Component>
    Not found
  </Component>
</Fragment>
`;
